checkout.py

```diff
@@ -4,13 +4,20 @@
 
 @author: vjham
 """
+from validate_clerk_input import validate_clerk_input
 
 items = []
+
+print("Enter 'DONE' at anytime to generate the final receipt.")
```

Comment:
This is a good change. BUT, I don't think it's really related to the rest of your changes. I would create a separate GitHub issue for this and then add a "TODO" here on this line, something like this:

TODO: <link to Github issue> Add explanatory text telling clerk to enter DONE

The reason I think this change should be in a separate branch / PR and not in this one is because I like to work with the worst case scenario in mind . Say for instance, several weeks go by and we decide that we don't want to validate clerk input anymore. So we roll back your changes (the changes in this PR) (by reverting the commit that merged this branch into `develop`). We don't want to roll back this message to the clerk as well! That is a good change that we would want to keep, even if we rolled back the code that validates clerk input. So we don't want to lump this change together with the validation changes.

Does that make sense?

Reply:
That makes sense! I am unsure of what action I should take now. Should I rectify this? I can delete the change, make a commit to my branch, and then submit a new pull request (after of course handling everything else you brought up). Then create the issue, make a new branch for that issue, and then make that change.

Reply:
ahh! Amazing question!!

This is what I would do (some of which you already said):

1. Delete this line from your code and then commit that change (if you had committed this change by itself originally, instead of lumping it in to a commit with unrelated changes, you could just revert the original commit or delete it from your branch's commit history. Interactive rebase is another option - that would make it look like you never made this change in the first place - if you want to try that, let me know and I will help!)

2. Push your updated vjh_14 up to GitHub

3. No need to submit a new PR. This PR will be automatically updated with any updates to the vjh_14 branch 

4. Create a new GitHub issue. But then just leave the issue for someone to do later. No need to rush and make a new branch and all that. Focus on getting this PR done first. 

YOU ARE DOING GREAT